packages/steveo/src/consumers/sqs.ts

```diff
@@ -66,15 +67,21 @@ class SqsRunner extends BaseRunner implements IRunner {
               await this.deleteMessage(c.topic, message);
             }
 
-            this.logger.debug('Start subscribe', c.topic, params);
             if (!task) {
               this.logger.error(`Unknown Task ${c.topic}`);
               return;
             }
 
-            await task.subscribe(params, runnerContext);
-            this.logger.debug('Completed subscribe', c.topic, params);
-            const completedContext = getContext(params);
+            const { context = {} } = c.payload;
+
+            this.logger.info(
+              { context, params },
+              `Start Subscribe to ${task.name}`
+            );
+
+            await task.subscribe(params, { ...context, ...runnerContext });
```

Comment:
Aren't we better of having `getContext` pull in the job information if available?

Reply:
yeah this

Reply:
@sugendran I think that's much cleaner